packages/terra-clinical-label-value-view/src/terra-dev-site/doc/clinical-label-value-view/clinicalLabelValueView.1.doc.mdx

```diff
@@ -11,8 +11,27 @@ import LabelValueViewPropsTable from 'terra-clinical-label-value-view/src/LabelV
 
 # Terra Clinical Label Value View
 
-The label value view component displays a label and the associated value or list of values underneath the label.
+The LabelValueView component creates a visual and programmatic association of a label-value pair. An HTML description list is used to create the programmatic structure of the label to a value. This semantic structure is critical to support users of assistive technology. 
 
+LabelValueView can be used as a single description list with one label-value pair or as a group of related label-value pairs.
```

Comment:
Can we add two short code examples in this doc to illustrate the different usages?

Reply:
I have already updated the examples in the previous Jira, to ilustrate the 2 different usage options of the LabelValueView - as a definition list or as individual term-definition pair. In the current PR, I have also set the aria-labelledby/ariaLabelledBy props for the existing examples.